tasks/allocs/src/lib.rs

```diff
@@ -0,0 +1,153 @@
+#![expect(clippy::print_stdout)]
+
+use std::{
+    fmt::Write as _,
+    fs::File,
+    io::{self, Write},
+};
+
+use humansize::{DECIMAL, format_size};
+
+use oxc_allocator::Allocator;
+use oxc_parser::{ParseOptions, Parser};
+use oxc_tasks_common::{TestFiles, project_root};
+
+use std::alloc::{GlobalAlloc, Layout};
+use std::sync::atomic::{AtomicUsize, Ordering::SeqCst};
+
+#[global_allocator]
+static GLOBAL: TrackedAllocator = TrackedAllocator;
```

Comment:
I wonder whether we can get https://docs.rs/dhat/latest/dhat working as well. Just a thought though, no changes required.